Makefile

```diff
@@ -142,3 +148,69 @@ ci-build: ci-test build build-docker-image build-all-platforms bundle-all-platfo
 
 ci-publish: ci-build push-docker-image
 	echo "ci-publish is going to be disabled. We are adopting Github actions"
+
+helm_chart_source_dir := deploy/charts/venafi-kubernetes-agent
+BINDIR := $(ROOT_DIR)/_bin
+
+.PHONY: generate-manifests
+generate-manifests: ## Generates jetstack.io_venaficonnections.yaml.
+generate-manifests: | $(NEEDS_GO) _bin/tools/yq
+	@echo "# DO NOT EDIT. Use 'make generate-manifests' to regenerate." >$(helm_chart_source_dir)/crd_bases/jetstack.io_venaficonnections.yaml
+	go run ./make/connection_crd >>$(helm_chart_source_dir)/crd_bases/jetstack.io_venaficonnections.yaml
+
+	@echo "# DO NOT EDIT. Use 'make generate-manifests' to regenerate." >$(helm_chart_source_dir)/templates/venafi-connection-crd.without-validations.yaml
+	_bin/tools/yq 'del(.. | ."x-kubernetes-validations"?) | del(.metadata.creationTimestamp)' $(helm_chart_source_dir)/crd_bases/jetstack.io_venaficonnections.yaml >>$(helm_chart_source_dir)/templates/venafi-connection-crd.without-validations.yaml
+
+	@echo "# DO NOT EDIT. Use 'make generate-manifests' to regenerate." >$(helm_chart_source_dir)/templates/venafi-connection-crd.yaml
+	_bin/tools/yq 'del(.metadata.creationTimestamp)' $(helm_chart_source_dir)/crd_bases/jetstack.io_venaficonnections.yaml >> $(helm_chart_source_dir)/templates/venafi-connection-crd.yaml
```

Comment:
These are leftover that I forgot to remove. They don't do anything and don't work.

I will keep them here to avoid the extra hassle of rebasing and such since there is a stacked PR on top of this one. I will remove this code in #556.

For context, we decided with Tim to manually copy the CRDs for now, and add the automation of the CRDs in #556.